Client/mods/deathmatch/logic/CClientRenderElementManager.h

```diff
@@ -47,6 +49,7 @@ class CClientRenderElementManager
     uint GetScreenSourceCount() { return m_uiStatsScreenSourceCount; }
```

Comment:
All these getters must be const

Reply:
because i didn't write these codes i perfer not to touch them
if you can fix it yourself

Reply:
fixed!